content/blog/mermaid-tina-progress.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: "Progress Update: TinaCMS + Mermaid"
+date: '2024-010-10T14:00:00.000Z'
+last_edited: '2024-010-10T14:00:00.000Z'
+author: Jack Pettit
+---
+
+The ability to integrate and manage Mermaid diagrams within TinaCMS is nearly here, offering more flexibility for content creators working with visual representations like flowcharts, sequence diagrams, and more. Here’s a look at where things stand with this feature and what’s coming next.
+
+### Part 1: Plate.js Integration (Completed)
+
+Mermaid diagrams are now fully integrated into the **Plate editor**. Using the `mermaid` package, diagrams can be edited and rendered directly within the editor. 
+
+![](/img/blog/mermaid-progress.gif)
+**Figure: Mermaid Diagram editing/Rendering inside the Rich Text Editor**
+
+### Part 2: Parsing and Saving Mermaid Diagrams (In Progress)
+
+The next phase focuses on **parsing and saving** these diagrams. Since TinaCMS uses customized `mdast`, the challenge lies in ensuring the Mermaid syntax is correctly parsed, saved, and reloaded without issues. 
+
+### What’s Next?
+
+Next steps include polishing the parsing and saving process and rolling out the final deployment. Once that’s done, you will be able to fully integrate, edit, and save Mermaid diagrams within TinaCMS 🦙
```

Comment:
```suggestion
Next steps include polishing the parsing and saving process and rolling out the final deployment. This will be coming in the 2.3.0 release of `tinacms`. Once that’s done, you will be able to fully integrate, edit, and save Mermaid diagrams within TinaCMS 🦙
```